sprout/sprout.sh

```diff
@@ -368,7 +369,7 @@ function start_flower() {
     cddir
     if [ -e "${PIDFILE_FLOWER}" ] ;
     then
-        ps -p `cat $PIDFILE_FLOWER` >/dev/null
+        ps -p "cat $PIDFILE_FLOWER" >/dev/null
```

Comment:
This looks slightly weird to me, since inside `""` it won't execute the `cat` ... or yes?